docs/Reference/Logging.md

```diff
@@ -22,21 +22,24 @@ const fastify = require('fastify')({
 ```
 
 Enabling the logger with appropriate configuration for both local development
-and production environment requires bit more configuration:
+and production and test environment requires bit more configuration:
+
 ```js
+const envToLogger = {
+  development: {
+    transport: {
+      target: 'pino-pretty',
+      options: {
+        translateTime: 'HH:MM:ss Z',
+        ignore: 'pid,hostname',
+      },
+    },
+  },
+  production: true,
+  test: false,
+}
 const fastify = require('fastify')({
-  logger: {
-      transport:
-        environment === 'development'
-          ? {
-              target: 'pino-pretty',
-              options: {
-                translateTime: 'HH:MM:ss Z',
-                ignore: 'pid,hostname'
-              }
-            }
-          : undefined
-    }
+  logger: envToLogger[environment] || true // defaults to true if no entry matches in the map
```

Comment:
When environment is `test`, it will be `true`.
`false || true = true`

Reply:
```suggestion
  logger: envToLogger[environment] && true // defaults to true if no entry matches in the map
```

Reply:
```suggestion
  logger: envToLogger[environment] ?? true // defaults to true if no entry matches in the map
```

Reply:
Good catch. Thanks